SimplifiedTests/NYPLContentDownloaderTests.swift

```diff
@@ -122,14 +125,14 @@ class NYPLContentDownloaderTests: XCTestCase {
       description: "NYPLAxisContentDownloader and NYPLAxisNetworkExecutor should deinitialize upon failure")
     expectation.expectedFulfillmentCount = 2
     
-    
-    var customContentDownloader: CustomNYPLContentDownloader? = CustomNYPLContentDownloader(networkExecuting: MockAxisNetworkExecutor())
+    var customContentDownloader: CustomNYPLContentDownloader? = CustomNYPLContentDownloader(
```

Comment:
minor: we shouldn't need `: CustomNYPLContentDownloader?`

Reply:
we're testing to make sure CustomNYPLContentDownloader gets deinitialized properly without any retain cycles or memory leaks by first creating an optional object and later nullifying it. Without the explicit `CustomNYPLContentDownloader?` we can't make it nil to test it.